packages/eds-core-react/.storybook/components/InfoCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { Card, Typography, Paper, Button } from '../../src'
+import { tokens } from '@equinor/eds-tokens'
+import styled from 'styled-components'
+
+const {
+  colors: {
+    infographic: {
+      primary__moss_green_100: { rgba: mossgreen100 },
+    },
+  },
+} = tokens
+
+type InfoCardTypes = {
+  title?: string
+  description?: string
+  url?: string
+  urlTitle?: string
+}
+
+const SBCard = styled(Card)`
```

Comment:
Maybe wait for the card changes and see if some stuff can be removed from here then

Reply:
Update with latest changes to `Card`